### PR TITLE
[SYNCOPE-1658] Tabular Topology

### PR DESCRIPTION
**Pull request by SamuelGaro**

### CODE REVIEW

**Comment by ilgrosso** [comment]
@SamuelGaro I see the following test failures:

```
Error:  Failures: 
Error:    AjaxBrowseITCase.browsingBookmarkablePageLink:64 path: 'body:idmPages:0:idmPageLI:idmPage' does not exist for page: Realms
Error:    BatchesITCase.executePropagationTask:356 path: 'body:idmPages:0:idmPageLI:idmPage' does not exist for page: Dashboard
Error:    TopologyITCase.login:46 path: 'body:idmPages:0:idmPageLI:idmPage' does not exist for page: Dashboard
Error:    TopologyITCase.login:46 path: 'body:idmPages:0:idmPageLI:idmPage' does not exist for page: Dashboard
Error:    TopologyITCase.login:46 path: 'body:idmPages:0:idmPageLI:idmPage' does not exist for page: Dashboard
Error:    TopologyITCase.login:46 path: 'body:idmPages:0:idmPageLI:idmPage' does not exist for page: Dashboard
Error:    TopologyITCase.login:46 path: 'body:idmPages:0:idmPageLI:idmPage' does not exist for page: Dashboard
Error:    TopologyITCase.login:46 path: 'body:idmPages:0:idmPageLI:idmPage' does not exist for page: Dashboard
Error:    TopologyITCase.login:46 path: 'body:idmPages:0:idmPageLI:idmPage' does not exist for page: Dashboard
Error:    TopologyITCase.login:46 path: 'body:idmPages:0:idmPageLI:idmPage' does not exist for page: Dashboard
Error:    TopologyITCase.login:46 path: 'body:idmPages:0:idmPageLI:idmPage' does not exist for page: Dashboard
Error:    TopologyITCase.login:46 path: 'body:idmPages:0:idmPageLI:idmPage' does not exist for page: Dashboard
Error:    TopologyITCase.login:46 path: 'body:idmPages:0:idmPageLI:idmPage' does not exist for page: Dashboard
```

**Comment by SamuelGaro** [comment]
> @SamuelGaro I see the following test failures:
> 
> ```
> Error:  Failures: 
> Error:    AjaxBrowseITCase.browsingBookmarkablePageLink:64 path: 'body:idmPages:0:idmPageLI:idmPage' does not exist for page: Realms
> Error:    BatchesITCase.executePropagationTask:356 path: 'body:idmPages:0:idmPageLI:idmPage' does not exist for page: Dashboard
> Error:    TopologyITCase.login:46 path: 'body:idmPages:0:idmPageLI:idmPage' does not exist for page: Dashboard
> Error:    TopologyITCase.login:46 path: 'body:idmPages:0:idmPageLI:idmPage' does not exist for page: Dashboard
> Error:    TopologyITCase.login:46 path: 'body:idmPages:0:idmPageLI:idmPage' does not exist for page: Dashboard
> Error:    TopologyITCase.login:46 path: 'body:idmPages:0:idmPageLI:idmPage' does not exist for page: Dashboard
> Error:    TopologyITCase.login:46 path: 'body:idmPages:0:idmPageLI:idmPage' does not exist for page: Dashboard
> Error:    TopologyITCase.login:46 path: 'body:idmPages:0:idmPageLI:idmPage' does not exist for page: Dashboard
> Error:    TopologyITCase.login:46 path: 'body:idmPages:0:idmPageLI:idmPage' does not exist for page: Dashboard
> Error:    TopologyITCase.login:46 path: 'body:idmPages:0:idmPageLI:idmPage' does not exist for page: Dashboard
> Error:    TopologyITCase.login:46 path: 'body:idmPages:0:idmPageLI:idmPage' does not exist for page: Dashboard
> Error:    TopologyITCase.login:46 path: 'body:idmPages:0:idmPageLI:idmPage' does not exist for page: Dashboard
> Error:    TopologyITCase.login:46 path: 'body:idmPages:0:idmPageLI:idmPage' does not exist for page: Dashboard
> ```

@ilgrosso I'm going to correct them

File: client/idrepo/console/src/main/java/org/apache/syncope/client/console/init/ClassPathScanImplementationLookup.java [hunk 1]
```diff
@@ -180,7 +185,13 @@ public void load() {
                         }
                     } else if (BasePage.class.isAssignableFrom(clazz)) {
                         if (clazz.isAnnotationPresent(IdMPage.class)) {
-                            idmPages.add((Class<? extends BasePage>) clazz);
+                            if (clazz.getName().endsWith("Topology")) {
```
Comment (ilgrosso) [inline client/idrepo/console/src/main/java/org/apache/syncope/client/console/init/ClassPathScanImplementationLookup.java:188]: Can you rewrite these nested conditions?

File: client/idrepo/console/src/main/java/org/apache/syncope/client/console/wizards/WizardMgtPanel.java [hunk 2]
```diff
@@ -228,12 +228,12 @@ public void onEvent(final IEvent<?> event) {
             } else if (event.getPayload() instanceof AjaxWizard.NewItemFinishEvent) {
                 SyncopeConsoleSession.get().success(getString(Constants.OPERATION_SUCCEEDED));
                 target.ifPresent(ajaxRequestTarget ->
-                    ((BaseWebPage) pageRef.getPage()).getNotificationPanel().refresh(ajaxRequestTarget));
+                        ((BaseWebPage) pageRef.getPage()).getNotificationPanel().refresh(ajaxRequestTarget));
```
Comment (ilgrosso) [inline client/idrepo/console/src/main/java/org/apache/syncope/client/console/wizards/WizardMgtPanel.java:231]: Please revert these formatting-only changes

**Comment by ilgrosso** [comment]
Thanks @SamuelGaro

**Pull request merged**